design/www/app/login/login.controller.js

```diff
@@ -0,0 +1,100 @@
+(function () {
+    'use strict';
+
+    angular.module('starter')
+            .controller('loginController', loginController);
+
+    function loginController($state, loginFactory, localStorageService, $localStorage, tostService, deviceService, googleLogin, facebookLogin) {
+        console.log('login');
+        this.data = {
+            email: '',
+            password: ''
+        }
+        // var deviceUUID = deviceService.getuuid();
+        var deviceUUID = 'device_id';
+        this.login = function () {
+            console.log(this.data.email);
+            console.log(this.data.password);
+            var query = loginFactory.save({
+                action_type: 'manual_login',
+                social_id: '',
+                platform: 'android',
+                token: 'token',
+                action: 'login_register',
+                device_id: 'dsvfdff',
+                email: this.data.email,
+                password: this.data.password,
+                name:''
+            });
+            query.$promise.then(function (data) {
+                console.log(data);
+                if(data.message == 'Please verify you account first'){
```

Comment:
this is wrong logic totally. if condition should never be on basis of message.
there should be a proper status: from api